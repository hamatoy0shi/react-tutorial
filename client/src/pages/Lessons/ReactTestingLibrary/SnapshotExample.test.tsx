/* eslint-disable import/no-extraneous-dependencies */
import React from 'react';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom';

function SomeComponent() {
  return (
    <div aria-label="test" id="some-component">
      <div aria-label="test">Название</div>
      <div>Описание</div>
    </div>
  );
}

describe('snapshot testing', () => {
  test('SomeComponent', () => {
    const { container } = render(<SomeComponent />);
    expect(container).toMatchSnapshot();
  });
});
