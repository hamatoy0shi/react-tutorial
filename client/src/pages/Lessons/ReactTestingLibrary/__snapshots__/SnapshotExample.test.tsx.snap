// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot testing SomeComponent 1`] = `
<div>
  <div
    aria-label="test"
    id="some-component"
  >
    <div
      aria-label="test"
    >
      Название
    </div>
    <div>
      Описание
    </div>
  </div>
</div>
`;
